src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -77,81 +90,28 @@ export interface ShippingState {
   phoneNumberTouched: PhoneNumberTouched
   addressErrors: AddressErrors
   addressTouched: AddressTouched
-  showEditModal: boolean
+  selectedSavedAddress: string
+  editAddressIndex: number
+  saveAddress: boolean
 }
 
 const logger = createLogger("Order/Routes/Shipping/index.tsx")
-
-const EditButton = styled(Text)`
-  &:hover {
-    text-decoration: underline;
-  }
-`
-
 @track()
 export class ShippingRoute extends Component<ShippingProps, ShippingState> {
```

Comment:
props on the refactor of this crazy component too! much simpler with a lof of the code pulled into other files.

Reply:
🙏🏽 I mostly moved the code around to make `index.tsx` smaller. Exactly what you wanted to do but we didn't have enough time 😄 
The only big change was that I [ended up](https://github.com/artsy/force/pull/6949/commits/f24ffa5673bdb43906b2d5855e6a0e54baecdab2) not updating the selected address in the `state` and just used that for when users type a new address in the form.